corehq/apps/style/decorators.py

```diff
@@ -1,27 +1,4 @@
 from functools import wraps
-from corehq.apps.style.utils import set_bootstrap_version3
-from crispy_forms.utils import set_template_pack
-
-
-def use_bootstrap3(view_func):
```

Comment:
@proteusvacuum you mean something like just leaving this function here as

```
def use_bootstrap3(view_func):
    warnings.warn("use_bootstrap3 is now defaut, please remove.", DeprecationWarning)
    return view_func
```

so that other branches that have this don't end up 500ing when merged?